src/navigators/Navigator.tsx

```diff
@@ -93,7 +94,7 @@ class Navigator extends React.Component<Props, State> {
     if (__DEV__) {
       return false;
     }
-    return !isAuthenticated && isTxPasswordSet && isPinSet;
+    return !isAuthenticated && isTxPasswordSet && isPinSet && AppState.currentState === 'active';
```

Comment:
does it allow the user to hack pin by clicking quickly on the tab bar?

Reply:
could not reproduce